src/components/SingleOrder.js

```diff
@@ -0,0 +1,41 @@
+import React, {useState, useEffect} from 'react';
+import { useParams } from 'react-router-dom';
+import { getAllOrders } from '../api/index.js';
+
+
+const SingleOrder = () => {
+    const {orderId} = useParams();
+    const [orders, setOrders] = useState([])
+    const fetchAllOrders = () => {
+      getAllOrders()
```

Comment:
We could use the single order API call instead of getting all of them... this could create an unnecessarily-long api call...